### PR TITLE
Juan.angarita/ch23066/integrations change wallet name and emoji

### PR DESCRIPTION
**Pull request by angarita-dev**
## Changelog

- Enabled WalletDetails view
- Integrated update principal method

### Type of changes included:

- [x] Components
- [x] Business Logic
- [ ] Bug fixes
- [x] Styling
- [ ] Code Refactor

### Clubhouse Tickets Related:

- Ticket 1
- Ticket 2
- Ticket 3

### Screenshots/Gifs:

https://user-images.githubusercontent.com/44899916/125534225-a00092e5-a41a-4385-a5b1-a9bd212234db.mov

### Notes:

*Place special notes here*

### Tested on:

- [x] Chrome
- [ ] Firefox
- [ ] Safari

### CODE REVIEW

File: source/Popup/Views/WalletDetails/index.jsx [hunk 1]
```diff
@@ -98,16 +134,24 @@ const WalletDetails = () => {
...
               )}
             />
           </Grid>
           <Grid item xs={12}>
-            <Button value={t('common.save')} variant="rainbow" fullWidth onClick={() => onSave()} />
+            <Button
+              value={t('common.save')}
+              variant="rainbow"
+              fullWidth
+              onClick={() => onSave()}
```
Comment (rocky-fleek) [inline source/Popup/Views/WalletDetails/index.jsx:152]: I think you can just pass ```onClick={onSave}```

File: source/Popup/Views/WalletDetails/styles.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  },
+  badge: {
+    marginRight: 12,
+    borderRadius: 6,
+    width: 'fit-content',
+    padding: '2px 8px',
+  },
+  accountBadge: {
+    background: '#D3E1FF',
+    color: '#3574F4',
```
Comment (rocky-fleek) [inline source/Popup/Views/WalletDetails/styles.js:21]: dont we have these colors somewhere?

**Review by rocky-fleek — APPROVED** [review]
LGTM, just some comments to improve code style

**Review by ChriLnth — APPROVED** [review]
LGTM

File: source/Popup/Views/WalletDetails/index.jsx (hunk 1 above)
Comment (angarita-dev) [reply source/Popup/Views/WalletDetails/index.jsx]: Added

File: source/Popup/Views/WalletDetails/styles.js (hunk 2 above)
Comment (angarita-dev) [reply source/Popup/Views/WalletDetails/styles.js]: I replaced with the values that we had on the theme

**Pull request merged**